exporter/datadogexporter/config.go

```diff
@@ -164,6 +167,14 @@ type SumConfig struct {
 	// The default is 'to_delta'.
 	// See https://docs.datadoghq.com/metrics/otlp/?tab=sum#mapping for details and examples.
 	CumulativeMonotonicMode CumulativeMonotonicSumMode `mapstructure:"cumulative_monotonic_mode"`
+
+	// The default is false, which submits sum as `count` type
+	// Enabling this changes the submission type from `count` to `rate`
```

Comment:
In Go, all documentation must start with the identifier.
```suggestion
	// SubmitAsRate reports whether the sum should be submitted as a rate. The default
	// is false, which means that it will be submitted as a count.
```